web/src/pages/project/AnalysisRunnerView/AnalysisRunnerSummary.tsx

```diff
@@ -80,20 +83,24 @@ const AnalysisRunnerSummary: React.FunctionComponent = () => {
         ])
     }
 
-    const flatData = data?.project.analyses.map(({ author, id, output, meta }, i) => ({
-        email: author,
-        id,
-        output,
-        ...meta,
-        position: i,
-        'Hail Batch': meta?.batch_url
-            ? meta?.batch_url.replace('https://batch.hail.populationgenomics.org.au/', '')
-            : '',
-        GitHub: `${meta?.repo}@${meta?.commit.substring(0, 7)}`,
-        Date: `${parseDate(sanitiseValue(meta?.timestamp))}`,
-        Author: `${parseEmail(sanitiseValue(author))}`,
-        Image: meta?.driverImage ? `${parseDriverImage(sanitiseValue(meta?.driverImage))}` : '',
-    }))
+    const flatData = data?.project.analyses.map(({ auditLogs, id, output, meta }, i) => {
+        const author = _.orderBy(auditLogs, ['timestamp'])[0]?.author
```

Comment:
Should this have `orders='desc'` to display the latest author and not the first?

Reply:
I want the first author, as that's the author that created the entry.

Reply:
Make sense